apps/backend/api/services/Frontend.js

```diff
@@ -145,16 +137,22 @@ module.exports = {
       return url(`/members/${getModelId(user)}`)
     },
 
-    post: function (post, group, isPublic, topic) {
+    post: function (post, group, extraParams = '') {
       const groupSlug = getSlug(group)
       let groupUrl = '/all'
 
-      if (isPublic) {
+      if (!group) {
         groupUrl = '/public'
       } else if (!isEmpty(groupSlug)) {
-        groupUrl = `/groups/${groupSlug}` + (topic ? `/topics/${topic}` : '')
+        const tags = post.relations.tags
+        const firstTopic = tags && tags.first()?.get('name')
+        if (firstTopic && (post.get('type') === Post.Type.CHAT || group.hasChatFor(firstTopic))) {
+          return url(`/groups/${groupSlug}/chat/${firstTopic}?postId=${post.id}&${extraParams}`)
```

Comment:
Is the main update needed for mobile that it needs to track `/chat/` related urls?

Reply:
yup

Reply:
Ok, I have that in a current branch